### PR TITLE
replace non-ascii characters in source code

### PR DESCRIPTION
**Pull request by kemen209**
This will generate warnings to block the building process of QGC. See [related bug in QGC](https://github.com/mavlink/qgroundcontrol/pull/9143)

### CODE REVIEW

File: src/sbf.h [hunk 1]
```diff
@@ -117,22 +117,22 @@
 #pragma pack(push, 1)
 
 typedef struct {
-	uint8_t  mode_type: 4;       /**< Bit ﬁeld indicating the PVT mode type, as follows:
-                                     0: No PVT available (the Error ﬁeld indicates the cause of the absence of the PVT solution)
+    uint8_t  mode_type: 4;       /**< Bit field indicating the PVT mode type, as follows:
```
Comment (bkueng) [inline src/sbf.h:120]: This shoud still use tabs.

**Review by bkueng — COMMENTED** [review]
Thanks, can you revert back to tabs for indentation? The style adheres to the [Autopilot](https://github.com/PX4/PX4-Autopilot).

**Comment by kemen209** [comment]
> Thanks, can you revert back to tabs for indentation? The style adheres to the [Autopilot](https://github.com/PX4/PX4-Autopilot).

Sorry for the inconvenience, i set my editor to automaticly convert tabs to spaces. Already revert all those space indentation to tabs.

**Review by bkueng — APPROVED** [review]
Thanks

**Pull request merged**